homeassistant/components/velux/__init__.py

```diff
@@ -46,13 +47,29 @@ class VeluxModule:
 
     def __init__(self, hass, config):
         """Initialize for velux component."""
-        from pyvlx import PyVLX
+        self._hass = hass
+        self._config = config
+        
```

Comment:
Initialize `self.pyvlx` to `None`. All instance attributes must be initialized in init method.